cdap-docs/admin-manual/source/installation/ambari/installing-cdap.rst

```diff
@@ -88,21 +88,12 @@ Installing CDAP
  
       **Ambari Dashboard:** Customizing Services 2
 
-#. If you wish to use the CDAP Explore service (to use SQL to query CDAP data), you must: have
-   Hive installed on the cluster; have the Hive client on the same host as CDAP; and set the
```

Comment:
I'm not sure this is a good idea. Why not just leave the image in, and then change the text to read:
```
and have the ``explore.enabled`` option set to true (the default).